### PR TITLE
has changed at line 75/76 FormPage

### PR DESCRIPTION
**Pull request by VictorDiniz89**
project: router.query.slug as string,
   //id: router.query.slug as string,

### CODE REVIEW

File: apps/dev-recruiters/src/components/modules/FormPage/signUpForm.tsx [hunk 1]
```diff
@@ -72,8 +72,8 @@ export default function SignUpForm() {
             .split(',')
             .map((skill) => ({ skill: skill })),
           role: router.query.position as string,
-          project: router.query.project as string,
-          id: router.query.slug as string,
+          project: router.query.slug as string,
+          //id: router.query.slug as string,
```
Comment (chungthuang) [inline apps/dev-recruiters/src/components/modules/FormPage/signUpForm.tsx:76]: @VictorDiniz89 I think we can delete this line since it's not used anymore.

**Comment by chungthuang** [comment]
Can you update the commit message to explain the functional changes, not describing the code?

File: apps/dev-recruiters/src/components/modules/FormPage/signUpForm.tsx (hunk 1 above)
Comment (VictorDiniz89) [reply apps/dev-recruiters/src/components/modules/FormPage/signUpForm.tsx:76]: Oh yep, I'll do that

**Comment by chungthuang** [comment]
@Enjoy2Live I shared https://github.com/dev-launchers/dev-launchers-platform/wiki#development-processes with Victor to remind him to target `staging/recruitment` branch next time. Since this PR is blocking migration to strapiv4, can we make it an exception to merge to master branch directly?

**Comment by Enjoy2Live** [comment]
> @Enjoy2Live I shared https://github.com/dev-launchers/dev-launchers-platform/wiki#development-processes with Victor to remind him to target `staging/recruitment` branch next time. Since this PR is blocking migration to strapiv4, can we make it an exception to merge to master branch directly?

ofc we can, there's the concept of hotfix (I haven't yet added it to docs) and I would consider this a hotfix so @VictorDiniz89 can already merge to master :smile:

**Pull request closed without merging**